performance-tests/pom.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright Kroxylicious Authors.
+
+    Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>kroxylicious-parent</artifactId>
+        <groupId>io.kroxylicious</groupId>
+        <version>0.2.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>io.kroxylicious</groupId>
+    <artifactId>performance-tests</artifactId>
+    <packaging>jar</packaging>
+
+    <name>performance-tests: microbenchmarks</name>
+
+    <dependencies>
```

Comment:
Would it make sense to have this module skip if 'quick' is activated?

            <activation>
                <property>
                    <name>!quick</name>
                </property>
            </activation>